lambdas/src/apis/third-party/controllers/third-party.ts

```diff
@@ -0,0 +1,22 @@
+import { Response } from 'express'
+import { TheGraphClient } from '../../../utils/TheGraphClient'
+import { TimeRefreshedDataHolder } from '../../../utils/TimeRefreshedDataHolder'
+import { ThirdPartyIntegration } from '../../collections/types'
+
+let thirdPartyIntegrationsCache: TimeRefreshedDataHolder<ThirdPartyIntegration[]>
+
+export function initCache(theGraphClient: TheGraphClient): void {
+  thirdPartyIntegrationsCache = new TimeRefreshedDataHolder(() => fetchThirdPartyIntegrations(theGraphClient), '1m')
```

Comment:
Will we querying the graph every one minute? Is this really necessary? Also, can this be configurable?

Reply:
I think it's aligned with block times and graph indexing (more less). It is not actively updating this data every 1m, just holding it as caching strategy.

We can add timer configuration for sure.

Reply:
Okay, can we increase that default interval? Or do you think that it's a risk?